src/main/java/org/icgc_argo/workflow/search/model/graphql/Analysis.java

```diff
@@ -29,16 +29,32 @@
 import lombok.SneakyThrows;
 
 @Data
-@AllArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class Analysis {
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
   private String analysisId;
 
+  private String studyId;
+
+  public Analysis(String analysisId, String studyId) {
+    if (null!=analysisId) {
+      this.analysisId = analysisId;
+    } else {
+      this.analysisId = "";
+    }
+    if (null!=studyId) {
+      this.studyId = studyId;
+    } else {
+      this.studyId = "";
+    }
+  }
+
```

Comment:
Can this be accomplished with `@AllArgsConstructor` and some annotations for default values? Its also possible that for the filters having null works better than empty string, might change the matching behavior.